ci/provisioning/post_provision_config_nodes_LEAP_15.sh

```diff
@@ -1,13 +1,14 @@
 #!/bin/bash
 
-REPOS_DIR=/etc/dnf.repos.d
+REPOS_DIR=/etc/dnf/repos.d
```

Comment:
We will need to do this differently in the future.
The code should not assume anything from the contents of the environment variables supplied by Jenkins or it will not be portable to other CI environments.

Reply:
`$REPOS_DIR` is not supplied by Jenkins.  It is being instantiated right here, in this file that is specific to Leap 15.

Reply:
I was referring to a reference to repo.dc.hpdd.intel.com that I thought I saw in an unchanged section of code.
I cannot find it in the current review.